#! /bin/sh /usr/share/dpatch/dpatch-run
## 08-fix-perl-shebang.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add missing Perl shebangs in examples.

@DPATCH@
--- ACE_wrappers.orig/examples/IPC_SAP/SOCK_SAP/summarize
+++ ACE_wrappers/examples/IPC_SAP/SOCK_SAP/summarize
@@ -1,3 +1,4 @@
+#!/usr/bin/perl
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
--- ACE_wrappers.orig/examples/IPC_SAP/SSL_SAP/summarize
+++ ACE_wrappers/examples/IPC_SAP/SSL_SAP/summarize
@@ -1,3 +1,4 @@
+#!/usr/bin/perl
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
--- ACE_wrappers.orig/bin/split-cpp
+++ ACE_wrappers/bin/split-cpp
@@ -1,6 +1,4 @@
-eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
-    & eval 'exec perl -w -S $0 $argv:q'
-    if 0;
+#!/usr/bin/perl
 
 # Splits C++ source files into one file per function or data item.
 #
